api/resources_portal/management/commands/material_additional_metadata_key_change.py

```diff
@@ -0,0 +1,183 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+
+from resources_portal.config.logging import get_and_configure_logger
+from resources_portal.models import Material
+
+logger = get_and_configure_logger(__name__)
+
+
+def validate_category(category):
+    return len([choice for choice in Material.CATEGORY_CHOICES if category in choice]) > 0
+
+
+def paginate_queryset(query_set, function, pagination):
+    current_page = 0
+    size = len(query_set)
+    current_slice = []
+    materials_updated = 0
+
+    while current_page < size:
+        if (size - current_page) < pagination:
+            current_slice = query_set[current_page : current_page + (size - current_page)]
+            current_page += size - current_page
```

Comment:
```suggestion
            current_page = size
```
`current_page + size - current_page` == `size`

Reply:
lol wow. I need to take some more courses in arithmetic. Done!